nestkernel/connection_manager.cpp

```diff
@@ -1122,7 +1116,13 @@ nest::ConnectionManager::get_connections( ArrayDatum& connectome,
 #ifdef _OPENMP
 #pragma omp critical( get_connections )
 #endif
-        connectome.append_move( conns_in_thread );
+        {
+          while ( !conns_in_thread.empty() )
```

Comment:
@heplesser, @jougs Thanks for review!

> In newly written NEST code, logical operators (!, &&, ÌI) should be written out asnot,and,or` for greater clarity.

Sure, will change it.

> I am also wondering if std::copy() wouldn't be faster and more expressive, although it would come at the price of temporarily having two copies of the connectivity data. But since the doubling is per-thread only, I think that would not be too bad.

Actually I did use std::copy() in the first version of the changes and I didn't notice any performance benefits of using it. Maybe it's faster but considering the whole method execution time it's not significant. So I decided that memory is more important than just nicely looking code. If you agree I will move this code to a separate method as you suggested.